src/test/regress/sql/upgrade_isolate_placement_before.sql

```diff
@@ -0,0 +1,21 @@
+SET client_min_messages TO WARNING;
```

Comment:
I this PR should also add tests to `src/test/regress/sql/shard_rebalancer_unit.sql`, to show how the rebalancer plans moves in relation with isolation of placements.